package.json

```diff
@@ -43,6 +44,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^10.2.1",
     "@testing-library/user-event": "^7.2.1",
-    "eslint-plugin-react-hooks": "^4.0.4"
+    "eslint-plugin-react-hooks": "^4.0.4",
```

Comment:
I love this idea, is this better served as a "DevDependencie" though?

Reply:
Github seemed to truncate the lines so it looked like this line was part of `dependencies` but this is currently a dev dependency.